platform_code/arduino/native_ethernet/micro_ros_transport.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include <micro_ros_platformio.h>
 
-#include <EthernetUdp.h>
+#include <NativeEthernetUdp.h>
```

Comment:
Is this valid for all Arduino frameworks?

Reply:
Can we use `#include <NativeEthernet.h>`, just as the transport used on [micro_ros_arduino](https://github.com/micro-ROS/micro_ros_arduino/blob/humble/src/native_ethernet_transport.cpp#L10).

Right now we only support the `native_ethernet` transport for the Teensy 4.1 board, which uses this library. 
We should be OK.

Reply:
In this case, this LGTM